docs/JSON-RPC.md

```diff
@@ -0,0 +1,435 @@
+
```

Comment:
Should be linked on the website.

Reply:
Do you mean that the file should be referenced in the website docs? I agree.
I have not looked into the website docs regarding this PR at all, maybe someone else who is already familiar with the history could handle that? @gilgongo / @ann0see?

Reply:
I tagged this PR as needing some docs, yes. Also discussed how that was going to be done somewhere in the comments I think.

Reply:
Yes. See https://github.com/jamulussoftware/jamuluswebsite/issues/717 I'll resolve this conversation now.